src/clients/rust/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "tigerbeetle"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+bitflags = "2.6.0"
+futures = "0.3.31"
```

Comment:
Just to double check, `futures` is our private dependency, and the public API is phrased strictly in terms of `async fn`, right? 


I think we could even depend just on `futures-channel` here?

Reply:
Yes it is an internal implementation detail. I have changed the dependency to futures-channel; we still depend on futures as a dev-dependency (for block_on).